src/Controller/Cliente/CadastrarClienteController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Controller\Cliente;
+
+use App\Entity\Cliente;
+use App\Repository\ClienteRepository;
+use App\Service\ValidarCpfService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+class CadastrarClienteController extends AbstractController
+{
+    private ClienteRepository $clienteRepository;
+    private ValidarCpfService $validarCpfService;
+
+    public function __construct(ClienteRepository $clienteRepository, ValidarCpfService $validarCpfService)
+    {
+        $this->clienteRepository = $clienteRepository;
+        $this->validarCpfService = $validarCpfService;
+    }
+
+    #[Route('/cliente/cadastrar/show', name: 'cadastrar_cliente_show')]
+    public function index(): Response
+    {
+        return $this->render('cliente/cadastrar_cliente.html.twig');
+    }
+
+    #[Route('/cliente/cadastrar', name: 'cadastrar_cliente')]
+    public function __invoke(Request $request, AuthenticationUtils $authenticationUtils): Response
```

Comment:
só para evitar problemas, o __invoke sempre é chamado quando a classe é chamada, ou seja esse vai ser o primeiro método e tals
```suggestion
    public function cadastrarAction(Request $request, AuthenticationUtils $authenticationUtils): Response
```